### DISCUSSION TITLE
Support kebab-case namespace alias

### DISCUSSION THREAD

**Discussion by alexdao3** [squint-cljs/cherry#70] [Ideas]
I think this is a bug, as I don't see any reason why this wouldn't be supported
```
;; cljs
["firebase-admin$default" :as fb-admin]
```

emits
```
;; mjs
import fb-admin from 'firebase-admin';
```

`fb-admin` is invalid as a Javascript variable due to hyphens

Error:
`SyntaxError: Unexpected token '-'`

**Comment by borkdude** [comment]
Not sure why this should be supported: CLJS and shadow-cljs don't support it either right?

**Reply by borkdude** [reply]
Sorry, I think I read the report wrong.

Do you mean:

emits `fb-admin` instead of emits `fb_admin` ?

**Reply by alexdao3** [reply]
Yes, sorry about that. I meant "emits `fb-admin`"

**Reply by alexdao3** [reply]
If you point me roughly to where this logic lives, I could try to PR a fix if you agree

**Reply by borkdude** [reply]
@alexdao3 Ah yes, sorry for the pre-mature conversion into a Discussion. I created an issue from this now again.

**Comment by martinklepsch** [comment]
This was fixed here: https://github.com/squint-cljs/cherry/issues/71

(Posting for anyone else who might be wondering)
